README.md

```diff
@@ -153,10 +162,13 @@ disables three-finger tapping. Defaults to 0.
 
 **ClickTime** - 
 When tapping, how much time to hold down the emulated button. Integer value
-representing milliseconds. Defaults to 50.
+representing milliseconds. Not very usable because button release event will be
```

Comment:
Phrasing. "Not very usable" is subjective. It would be better to point out in what cases is it useful to change this value.